i18n/locales/ca.json

```diff
@@ -35,5 +35,6 @@
   "components.pagination.next": "Següent | ",
   "components.pagination.previous": "Anterior",
   "layouts.blogPost.author.byLine": "{author, select, null {} other {By {author}, }}",
-  "layouts.blogIndex.currentYear": "News from {year}"
+  "layouts.blogIndex.currentYear": "News from {year}",
+  "components.blog.blogCard.author.by": "by"
```

Comment:
You should only migrate the English translation key :)

Can you remove all other translations, pretty please 🙇